cpp/src/arrow/c/abi.h

```diff
@@ -80,6 +80,24 @@ struct ArrowArray {
   void* private_data;
 };
 
+#  define ARROW_STATISTICS_KEY_AVERAGE_BYTE_WIDTH_EXACT "ARROW:average_byte_width:exact"
```

Comment:
Don't know constexpr std::string_view is better or this is better

Reply:
We can't use `constexpr` because this header may be used by C programs.